Description: Upstream added functionality in the open file dialog for
 selected audio and video files. Extended to work on case sensitive
 systems.
Origin: upstream, http://code.google.com/p/winff/source/detail?r=584
Bug: http://code.google.com/p/winff/issues/detail?id=140
Author: Paul Gevers <paul@climbing.nl>

Index: winff-git/unit1.lfm
===================================================================
--- winff-git.orig/unit1.lfm	2011-12-11 15:45:47.181455407 +0100
+++ winff-git/unit1.lfm	2011-12-11 15:47:17.493903241 +0100
@@ -2118,6 +2118,7 @@
   end
   object dlgOpenFile: TOpenDialog
     Title = 'Select Video Files...'
+    Filter = 'All Files|*.*|Audio Video Interleave|*.avi;*.AVI|MPEG|*.mpg;*.MPG;*.mpeg;*.MPEG|MOV|*.mov;*.MOV|Matroska|*.mkv;*.MKV|DVD Vob|*.vob;*.VOB|MP3|*.mp3;*.MP3|OGG Theora|*.ogg;*.OGG|WEBM|*.webm;*.WEBM'
     FilterIndex = 0
     Options = [ofAllowMultiSelect, ofEnableSizing, ofViewDetail]
     left = 520
